pkg/cmd/render/render.go

```diff
@@ -377,6 +382,42 @@ func (t *TemplateData) setPlatform(infraConfigFilePath string) error {
 	return nil
 }
 
+func (t *TemplateData) setUnsupportedKeyForCR(etcdOperatorCR string) error {
+	type UnSupported struct {
+		UseUnsupportedUnsafeNonHANonProductionUnstableEtcd bool `json:"useUnsupportedUnsafeNonHANonProductionUnstableEtcd"`
+	}
+	etcdOperatorCRStructure := &operatorv1.MyOperatorResource{}
+	etcdOperatorCRData, err := ioutil.ReadFile(etcdOperatorCR)
+	if err != nil {
+		return err
+	}
+	etcdJson, err := yaml.YAMLToJSON(etcdOperatorCRData)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(etcdJson, etcdOperatorCRStructure)
+	if err != nil {
+		return err
+	}
+	m := UnSupported{true}
+	b, err := json.Marshal(m)
+	if err != nil {
+		return err
+	}
+	etcdOperatorCRStructure.Spec.UnsupportedConfigOverrides = runtime.RawExtension{Raw: b}
+
+	updateEtcdData, err := json.MarshalIndent(etcdOperatorCRStructure, "", "  ")
+	if err != nil {
+		return err
+	}
+
+	dstFile := fmt.Sprintf("/usr/share/bootkube/manifests/manifests/%s", filepath.Base(etcdOperatorCR))
```

Comment:
not sure how this would work, I think the idea is to write a new manifest file in an output directory[1] and then override it in bootkube by a cp command to the global manifest directory.

tldr: if you can output it in assets-output[2] directory, the installer will copy the manifest in the right place[1]

1. https://github.com/openshift/installer/blob/75738a342c1973121eedda7d91096d21c19194c9/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L139
2. https://github.com/openshift/installer/blob/75738a342c1973121eedda7d91096d21c19194c9/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L132